src/shared/clients/s3Client.ts

```diff
@@ -264,6 +272,20 @@ export class DefaultS3Client {
         getLogger().debug('DownloadFile succeeded')
     }
 
+    public async getSignedUrl(request: SignedUrlRequest): Promise<string> {
+        const time = request.time ? request.time : 15
```

Comment:
Don't use a default. There should be no reason a user is getting to this point with an ambiguous time value.

Reply:
Agreed. If you every assign a default in code, do it once. Other functions should not need to know about these default values.